bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueBridge.java

```diff
@@ -851,6 +852,38 @@ public void deleteSchedule(Schedule schedule) throws IOException, ApiException {
         handleErrors(result);
     }
 
+    /**
+     * Returns the list of scenes that are not recyclable.
+     *
+     * @return all scenes that can be activated
+     */
+    public List<Scene> getScenes() throws IOException, ApiException {
+        requireAuthentication();
+
+        Result result = http.get(getRelativeURL("scenes"));
+        handleErrors(result);
+
+        Map<String, Scene> sceneMap = safeFromJson(result.getBody(), Scene.GSON_TYPE);
+        return sceneMap.entrySet().parallelStream()//
+                .map(e -> {
+                    e.getValue().setId(e.getKey());
+                    return e.getValue();
+                })//
+                .filter(scene -> !scene.isRecycle())//
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Activate scene to all lights that belong to the scene.
+     *
+     * @param id the scene to be activated
+     * @throws IOException if the bridge cannot be reached
+     */
+    public CompletableFuture<Result> activateScene(String id) {
```

Comment:
Just a minor remark: would `runScene` not be a better name ?

Reply:
How about recallScene? That would be consistent to the hue API